packages/autocomplete/src/elements/Autocomplete.js

```diff
@@ -0,0 +1,312 @@
+                    {!isOpen && (
+                      <StyledValueWrapper>{optionDictionary[selectedValue]}</StyledValueWrapper>
+                    )}
+                    <StyledInput
+                      {...getInputProps(
+                        getFieldInputProps(
+                          {
+                            bare: true,
+                            autocomplete: 'off',
+                            autocapitalize: 'off',
+                            autocorrect: 'off',
+                            innerRef: ref => {
+                              this.inputRef = ref;
+                              triggerInputRef(ref);
+                              inputRef && inputRef(ref);
+                            },
+                            value: inputValue,
+                            isOpen,
+                            placeholder,
+                            onChange: e => {
+                              this.setState({ inputValue: e.target.value });
+                            },
+                            onFocus: () => {
+                              this.setState({ isFocused: true });
+                            },
+                            onBlur: () => {
+                              this.setState({ isFocused: false });
+                            },
+                            onKeyDown: e => {
+                              if (
+                                e.keyCode === KEY_CODES.ENTER &&
+                                (!e.target.value || e.target.value.trim().length === 0) &&
+                                !focusedKey &&
+                                isOpen
+                              ) {
+                                e.preventDefault();
+                              }
+                            }
+                          },
+                          { isDescribed: false }
+                        )
+                      )}
+                    />
+                  </StyledFauxInput>
+                );
+              }}
+            >
+              {({ getMenuProps, getItemProps, placement }) => {
+                const props = getMenuProps({
+                  placement,
+                  animate: true,
+                  small,
+                  style: {
+                    width: this.wrapperRef ? this.wrapperRef.getBoundingClientRect().width : 0
+                  }
+                });
+
+                props.children = this.renderMenuItems(getItemProps);
+
+                if (renderDropdown) {
+                  return renderDropdown(props);
+                }
+
+                const { children: menuChildren, ...otherMenuProps } = props;
+
+                return (
+                  <MenuView {...otherMenuProps}>
+                    <StyledMenuOverflow maxHeight={maxHeight}>{menuChildren}</StyledMenuOverflow>
+                  </MenuView>
+                );
+              }}
+            </AutocompleteContainer>
+            {message}
```

Comment:
So we should be wrapping this in `<Message>` like we do with `<Hint>` and `<Label>` correct? We should also forward the validation prop here too

Reply:
Correct. Missed this in the first run-through. Now fixed.